tests/Database/DatabaseEloquentCollectionTest.php

```diff
@@ -202,14 +202,6 @@ public function testCollectionReturnsUniqueItems()
 	}
 
 
-	public function testLists()
```

Comment:
Hmmm. I wasn't saying we should add a crap load more tests, I'm just saying that removing the test isn't doing any good, while keeping it may be useful to catch future mess ups.